app-scroll-effects/app-scroll-effects.html

```diff
@@ -146,24 +178,27 @@
         fx.title.style.opacity = 1;
         fx.condensedTitle.style.opacity = 0;
       }
-      interpolate(Math.min(1, p), [
-        [1,  fx.scale],
-        [0, -fx.titleDX],
-        [y, y-fx.titleDY]
-      ],
-      function(scale, translateX, translateY) {
-        this.transform('translate(' + translateX + 'px, ' + translateY + 'px) ' +
-            'scale3d(' + scale + ', ' + scale + ', 1)',
-            fx.title);
-      }, this);
+      interpolate(Math.min(1, p),
+        [
+          [1,  fx.scale],
+          [0, -fx.titleDX],
+          [y, y-fx.titleDY]
+        ],
+        function(scale, translateX, translateY) {
+            this.transform('translate(' + translateX + 'px, ' + translateY + 'px) ' +
+                'scale3d(' + scale + ', ' + scale + ', 1)',
+                fx.title);
+        },
+        this);
     }
-  };
+  },
 
-  /**
-   * Upon scrolling past a threshold, CSS transition the font size of a designated title element
-   * between two values.
-   */
-  scrollEffects['resize-snapped-title'] = {
+  'resize-snapped-title': {
+    /**
+     * Upon scrolling past a threshold, CSS transition the font size of a designated title element
+     * between two values.
```

Comment:
The placement of this comment is not consistent with the other effects, which are above the name of the effect.